test/index.test.js

```diff
@@ -716,6 +716,53 @@ Previous tag: ''
           payload: require('./fixtures/push')
         })
 
+        expect.assertions(1)
+      })
+    })
+    describe('custom replacers', () => {
```

Comment:
Could you add tests with multiple (maybe overlapping?) replacers as well?

Reply:
Its easier to set them up in template.test, so I did it there